<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AbstrctForm</class>
 <widget class="QWidget" name="AbstrctForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>255</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="image_label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>300</width>
     <height>225</height>
    </rect>
   </property>
   <property name="text">
    <string>图片显示</string>
   </property>
  </widget>
  <widget class="QLabel" name="start_time_label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>225</y>
     <width>150</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>开始时间:</string>
   </property>
  </widget>
  <widget class="QLabel" name="end_time_label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>240</y>
     <width>150</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>结束时间:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="play_button">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>225</y>
     <width>40</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>播放</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_abstract_index">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>230</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>摘要</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
